bitcoin/src/blockdata/block.rs

```diff
@@ -481,7 +485,7 @@ mod tests {
         // [test] TODO: check the transaction data
 
         assert_eq!(real_decode.size(), some_block.len());
-        assert_eq!(real_decode.strippedsize(), some_block.len());
```

Comment:
nit: This probably should have been changed to `stripped_size` in comit: `e7e60fde Rename strippedsize to stripped_size`

Reply:
Thanks! fixed.